### PR TITLE
feat: sp services add pprof

### PR DESCRIPTION
**Pull request by sysvm**
### Description

SP adds pprof function for optimizing service performance

### Rationale

N/A

### Example

N/A

### Changes

N/A

### CODE REVIEW

File: cmd/storage_provider/init.go [hunk 1]
```diff
@@ -91,6 +91,24 @@ func initResourceManager(ctx *cli.Context) error {
 	return nil
 }
 
+// initPprof initializes global pprof service for performance monitoring.
+func initPprof(ctx *cli.Context, cfg *config.StorageProviderConfig) error {
```
Comment (will-2012) [inline cmd/storage_provider/init.go:95]: Pls normalize `PProf` and `Pprof` into one style.

Comment (sysvm) [reply cmd/storage_provider/init.go]: fixed

File: model/const.go [hunk 2]
```diff
@@ -30,6 +30,8 @@ var (
 	P2PService = strings.ToLower("p2p")
 	// AuthService defines the name of auth service
 	AuthService = strings.ToLower("auth")
+	// PprofService defines the name of metrics service
+	PprofService = strings.ToLower("pprof")
```
Comment (will-2012) [inline model/const.go:34]: Rename to `PProfService`?

File: config/config.go [hunk 3]
```diff
@@ -151,11 +154,19 @@ var DefaultLogConfig = &LogConfig{
...
 var DefaultP2PConfig = &p2p.NodeConfig{
 	ListenAddress: model.P2PListenAddress,
 	PingPeriod:    model.DefaultPingPeriod,
 }
 
+// DefaultPProfConfig defines the default configuration of pprof service
+var DefaultPProfConfig = &pprof.PProfConfig{
+	Enabled:     false,
+	HTTPAddress: model.PProfHTTPAddress,
+}
```
Comment (will-2012) [inline config/config.go:167]: Add this conf to config_template.toml

File: model/const.go (hunk 2 above)
Comment (sysvm) [reply model/const.go:34]: fixed

File: config/config.go (hunk 3 above)
Comment (sysvm) [reply config/config.go:167]: fixed

**Review by xibao-nr — APPROVED** [review]

**Review by will-2012 — APPROVED** [review]

**Review by RenRick — APPROVED** [review]

**Pull request merged**